.appveyor.yml

```diff
@@ -13,6 +13,7 @@ init:
     - SET "SYMFONY_REQUIRE=>=4.2"
     - SET ANSICON=121x90 (121x90)
     - REG ADD "HKEY_CURRENT_USER\Software\Microsoft\Command Processor" /v DelayedExpansion /t REG_DWORD /d 1 /f
+    - SET MESSENGER_DOCTRINE_DSN=sqlite://\%APP_DATA%\db.sqlite
```

Comment:
the failures on appveror show that the test cases are missing an `@requires extension pdo`